examples/fastlib/tasks/retry_task_example.py

```
@@ -0,0 +1,29 @@
+    elif 33 < value <= 66:
+        raise YouCanTryAgain
+    else:
+        raise Exception('game over! :(')
+
+
+@task(queue_url=QUEUE_URL, region_name='us-east-1', max_retries=2)
+async def dummy_retry_task(message) -> None:
+    try:
+        test_your_luck(message)
+    except YouCanTryAgain:
+        raise RetryTask
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)